doc/changelog.rst

```diff
@@ -162,6 +162,12 @@ Breaking Changes in 4.0
   projection will now return the entire document, not just the ``"_id"`` field.
 - ``MongoClient()`` now raises a :exc:`~pymongo.errors.ConfigurationError`
   when more than one URI is passed into the ``hosts`` argument.
+- ``MongoClient()`` now raises an exc:`~pymongo.errors.InvalidURI` exception
+  when it encounters unescaped percent signs in username and password.
+- ``MongoClient()`` now uses `urllib.parse.unquote` rather than
```

Comment:
Here and everywhere else, MongoClient() should be an actual link to the class. 

Also `urllib.parse.unquote` and other methods should be actual links, eg:
```
:py:class:`urllib.parse.unquote`
```

Reply:
We also should make it clear that this only applies when parsing user info from a URI.